tgstation.dme

```diff
@@ -1995,6 +1996,22 @@
 #include "code\modules\hydroponics\grown\tobacco.dm"
 #include "code\modules\hydroponics\grown\tomato.dm"
 #include "code\modules\hydroponics\grown\towercap.dm"
+#include "code\modules\instruments\instrument_data\_instrument_data.dm"
+#include "code\modules\instruments\instrument_data\_instrument_key.dm"
+#include "code\modules\instruments\instrument_data\brass.dm"
+#include "code\modules\instruments\instrument_data\chromatic_percussion.dm"
+#include "code\modules\instruments\instrument_data\fun.dm"
+#include "code\modules\instruments\instrument_data\guitar.dm"
+#include "code\modules\instruments\instrument_data\hardcoded.dm"
+#include "code\modules\instruments\instrument_data\organ.dm"
+#include "code\modules\instruments\instrument_data\piano.dm"
+#include "code\modules\instruments\instrument_data\synth_tones.dm"
+#include "code\modules\instruments\instruments\item.dm"
+#include "code\modules\instruments\instruments\stationary.dm"

```

Comment:
This second `instruments` directory seems very arbitrary. I'd just move the files up a level tbh

Reply:
sure